.github/workflows/scripts/publish-sdkman.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
I suggest adding some comments at the top of the file explaining what the script intention is, and what are its requirements.